### PR TITLE
Create space-travelers-hub project

### PR DESCRIPTION
**Pull request by yonas44**
Hello👋, for this project we @yonas44 and @ifeanyi305 implemented:
- Fetch data from an API and update the store.
- Render rockets and mission components using the data from the store.
- Add routing for the navigation links.
- Add dispatch actions to update the state/ store with user interactions.
- Implement unit and integration testing using react testing library.
- Add mobile responsiveness for the app.

Thank you for reviewing our PR!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        <Route element={<Navbar />} exact path="/">
+          <Route element={<RocketList />} path="/" />
+          <Route element={<MissionsPage />} path="/missions" />
+          <Route element={<ProfilePage />} path="/profile" />
+          <Route element={<MissionsPage />} path="/missions" />
```
Comment (mirouhml) [inline src/App.js:17]: - [ ] I am not sure if this is a mistake but the route `/missions` is repeated twice, kindly remove it if it's a mistake.

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @yonas44 @ifeanyi305 , 

Good job so far, I really loved your website, especially the design!

There is only one issue that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (yonas44) [reply src/App.js:17]: Thank you @mirouhml for your feedback🙏, we will definitely have it removed😅👍

**Review by ichala — APPROVED** [review]
Hi @yonas44   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**